tests/testNRyield.cc

```diff
@@ -49,20 +55,31 @@ void testNRyield(G4double Emin, G4double Emax, const G4String& unit,
   if (!goodInput) ::exit(1);		// If anything failed, abort
 
   // Instantiate single instance of each of the named yield functions
-  const char* useNIEL[] = { "Lindhard", "LewinSmith", "Sarkis", "Impact" };
+  const char* useNIEL[] = {"Lindhard", "Empirical", "LewinSmith", "Sarkis", "Impact" };
   const size_t nNIEL = sizeof(useNIEL)/sizeof(char*);
 
   // NOTE: Can't use G4CMPConfigManager to do this mapping, because it
   //       deletes the previous pointer when a new one is requested.
+     
   const G4VNIELPartition* NIELfunc[nNIEL] = {
-    new G4CMPLindhardNIEL, new G4CMPLewinSmithNIEL,
-    new G4CMPSarkisNIEL, new G4CMPImpactTunlNIEL };
-
+    new G4CMPLindhardNIEL, new G4CMPEmpiricalNIEL(),
+    new G4CMPLewinSmithNIEL, new G4CMPSarkisNIEL,
+    new G4CMPImpactTunlNIEL };
+  
   // Output will be tab-delimited columns for all the NIEL functions
   G4cout << std::left << "Energy";
   for (size_t i=0; i<nNIEL; i++) G4cout << "\t" << std::setw(8) << useNIEL[i];
   G4cout << G4endl;
 
+  // Set Empirical lindhard parameters 
+  G4CMPEmpiricalNIEL* EmpNIEL = new G4CMPEmpiricalNIEL();
```

Comment:
You should not be re-instantiating a different instance here.  You instantiated the model at line 65.  Use a cast so you can reference the array element:
```
G4CMPEmpiricalNIEL* EmpNIEL = (G4CMPEmpiricalNIEL*)useNIEL[1];
```

Alternatively, you can move your current line 75 to be _before_ line 65, and then in the array initializer, put "EmpNIEL" in place.